addons/hw_drivers/static/src/js/worker.js

```diff
@@ -41,9 +42,11 @@
 
                         // Here we execute the code coming from the pos, apparently $.parseHTML() executes scripts right away,
                         // Since we modify the dom afterwards, the script might not have any effect
+                        /* eslint-disable no-undef */
                         if (typeof foreign_js !== 'undefined' && $.isFunction(foreign_js)) {
                             foreign_js();
                         }
+                        /* eslint-enable no-undef */
```

Comment:
Maybe report this to the eslint folks? `typeof <undef>` is supposed to be supported, but it's a bit dumb if you can't then used the symbol you just checked existed.

Though beforehand maybe try nesting the `isFunction` call in the conditional in case eslint just doesn't like the `&&`?